### ISSUE TITLE
`omicron-package package` depends on directories it doesn't create

### ISSUE THREAD

**Issue by davepacheco** [oxidecomputer/omicron#1299]
I'm not sure this is a bug but I found it surprising:

```
$ cargo run --bin=omicron-package package
...
Error: failed to create clickhouse in "out"

Caused by:
    failed to canonicalize "out/clickhouse": No such file or directory (os error 2)
$
```

Context: I was trying to follow the how-to-simulated instructions to run a Nexus with console assets.  The easiest way I thought to get the console assets was to run this.

**Comment by davepacheco** [comment]
It also needed `out/console-assets` to exist.

**Comment by davepacheco** [comment]
It next complained about `out/certs`.  Once I created that, I got a new problem that I haven't figured out how to work around:

```
Error: failed to create omicron-nexus in "out"

Caused by:
    failed to canonicalize "out/certs/cert.pem": No such file or directory (os error 2)
```

Guessing that maybe it was wanting `out/omicron-nexus`, I created that, but it didn't help.

**Comment by smklein** [comment]
These directories in `out` are normally created by the `./tools/install_prerequisites.sh` script (with the exception of the certificates, which could have been created with `./tools/create_self_signed_cert.sh` - though that's no longer relevant with https://github.com/oxidecomputer/omicron/pull/1300 ).

I'll update the error message to try to make this more clear.

**Comment by smklein** [comment]
I tried to update the documentation for that here: https://github.com/oxidecomputer/omicron/pull/1287/files#diff-3e8e855771db92d49870acadd204a2479d1fd60139d153347d1521d9d1def37c 

Regardless, that change has been reverted on the latest main, so certificates should no longer be required.

**Comment by davepacheco** [comment]
Got it -- so maybe the fix here is #1306 plus when we re-do the TLS change, add a hint for that?
